core/translations/cs.json

```diff
@@ -566,7 +565,10 @@
     "recovery__title_dry_run": "Kontrola zálohy",
     "recovery__title_recover": "Obnovit peněženku",
     "recovery__title_remaining_shares": "Zbývající části",
+    "recovery__title_unlock_repeated_backup": "UNLOCK BACKUP",
     "recovery__type_word_x_of_y_template": "Zadejte slovo {0} z {1}",
+    "recovery__unlock_repeated_backup": "Do you want to unlock the backup?",
+    "recovery__unlock_repeated_backup_verb": "Unlock backup",
```

Comment:
They are "inherited" from `en.json` in the sense that it will show english if the czech translation for the specific key is missing --- so these things are not necessary (so I would avoid them)

Reply:
However, they seem to be needed for tests, otherwise lookups during tests will fail.

Reply:
fix the lookup in tests then? :woman_shrugging: